.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
(I usually add these to my global dictionary, not sure they should be in the repo).